ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -37,7 +37,15 @@ export function GridLayout(props: GridLayoutProps) {
     ...others
   } = props;
 
-  const [isOpen, setIsOpen] = useState(!!spec.display?.collapse?.open);
+  // allow display to be optional, default to open state
+  const rowDisplay = spec.display ?? {
+    title: '',
+    collapse: {
+      open: true,
+    },
+  };
+
+  const [isOpen, setIsOpen] = useState(rowDisplay.collapse?.open ?? true);
```

Comment:
Was it just this state initializer (i.e. the `!!`) causing the issue? Feels weird to default the whole `display` property and show a title if there isn't one.

Reply:
(We're gonna want to support groups without titles eventually anyway)